Source/Plugins/EditorModules/SceneView/Modules/SceneView.cs

```diff
@@ -171,17 +176,10 @@ protected override void OnClosed(EventArgs e)
 			Scene.ComponentAdded -= this.Scene_ComponentAdded;
 			Scene.ComponentRemoving -= this.Scene_ComponentRemoving;
 		}
-
-		internal void SaveUserData(XElement node)
-		{
-			node.SetElementValue("ShowComponents", this.buttonShowComponents.Checked);
-		}
-		internal void LoadUserData(XElement node)
+		
```

Comment:
Style consistency: Another drive-by fix, this one should be a double newline as well.